tests/TimeKeeping/test_TimeKeeping.py

```diff
@@ -233,7 +233,6 @@ def test_str(self):
 
         atts_list = ['missionStart', 'missionPortion', 'missionLife', 'missionFinishAbs', 'currentTimeNorm',
                      'currentTimeAbs', 'OBnumber', 'OBduration', 'OBstartTimes', 'OBendTimes', 'cachedir']
-
```

Comment:
if possible, avoid doing changes like this (I randomly do this all the time and Dmitry complains simply for the fact that there is an extra file to review. I'd merge this, just avoid doing it.)